README.md

```diff
@@ -49,3 +49,52 @@ Create a kubernetes manifest for a pod which will containa ToDo app container:
 1. `README.md` Should have explained your strategy configuration (Why such numbers)
 1. `README.md` Should have explained how to access the app after deployment
 1. Create PR with your changes and attach it for validation on a platform.
+
+
+
+### how to deploy the app to k8s
+
+Use these commands:
+
+1) kubectl apply -f deployment.yml
+
+2) kubectl apply -f hpa.yml
+
+3) kubectl apply -f namespace.yml  # Create the namespace
```

Comment:
Incorrect order of this step. Also, this command will create todoapp namespace and that will cause an error.